src/shared/restore_status_view.rs

```diff
@@ -0,0 +1,33 @@
+use makepad_widgets::*;
+
+live_design! {
+    use link::theme::*;
+    use link::shaders::*;
+    use link::widgets::*;
+    use crate::shared::styles::*;
+
+    pub RestoreStatusView = <View>{
```

Comment:
based on my other comments, you'll need to make this a widget:

```suggestion
    pub RestoreStatusView = {{RestoreStatusView}} {
```